src/components/SettingsDialog/BoardSettings/BoardSettings.tsx

```diff
@@ -29,22 +32,128 @@ export const BoardSettings = () => {
   const [password, setPassword] = useState<string>("");
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmationDialog, setShowConfirmationDialog] = useState<boolean>(false);
+  const [isProtectedOnOpen, setIsProtectedOnOpen] = useState(state.board.accessPolicy === "BY_PASSPHRASE");
```

Comment:
The naming could be better I think since I don't know exactly what's meant with `isProtectedOnOpen`

Reply:
I couldnt think of a better name, any suggestions?

Reply:
Maybe `isProtectedOnInitialOpen` or `isProtectedOnInitialSettingsOpen`?